roles/libvirt_manager/tasks/create_cloud_init_iso.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Generate cloud-init iso
+  become: false
+  vars:
+    _images_dir: "{{ [ cifmw_libvirt_manager_basedir, 'workload', ] | ansible.builtin.path_join }}"
+  block:
+    - name: "Acquire authorized_keys from hypervisor"
+      ansible.builtin.slurp:
+        src: "{{ ansible_user_dir ~ '/.ssh/authorized_keys' }}"
+      register: _authorized_keys_b64
+
+    - name: Create a list of ssh pub keys from authorized_keys
+      ansible.builtin.set_fact:
```

Comment:
Setting this as a fact makes me think it's used multiple times but it's used only in the man set_fact below. Why aren't we merging this with the next other set_fact? Clarity?

Reply:
Clarity mainly, yep, but we can merge it since I don't have a strong opinion about it. I changed idea (at least) 3 times.

Reply:
ack, makes sense